### PR TITLE
data: project addition by Akshatchaube01

### PR DESCRIPTION
**Pull request by Akshatchaube01**
## Related Issue

issue #412 

## Description

I have added my open source project in the website 

## Screenshots

image

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -689,5 +689,23 @@
...
     ]
+  },
+  {
+    "github_username": "Akshatchaube01",
+    "Social_media": {
+      "gitHub": "https://github.com/Akshatchaube01",
+      "LinkedIn": "",
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": ""
```
Comment (priyankarpal) [inline src/DB/projects.json:700]: add your social media's link here

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Akshatchaube01** [comment]
I am not able to commit my changes, i am getting this error , can you please help me out , I am not trying to run it in my local machine 
image

**Comment by priyankarpal** [comment]
> I am not able to commit my changes, i am getting this error , can you please help me out , I am not trying to run it in my local machine image

use `pnpm format` to fix this problem & solve the conflict error.

File: src/DB/projects.json (hunk 1 above)
Comment (Akshatchaube01) [reply src/DB/projects.json]: social media links added @priyankarpal please review again

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**